### ISSUE TITLE
Request to upload a pretrained model

### ISSUE THREAD

**Issue by absurdvision** [zhr1201/CNN-for-single-channel-speech-enhancement#3]
Hi
Would it be possible to upload a pretrained model.
It would be very helpful to have a starting/reference model to further tune the network.

**Comment by zhr1201** [comment]
Sorry but it exceeds GitHub's file size limit of 100.00 MB.

**Comment by absurdvision** [comment]
Would you be able to upload to Google drive, Dropbox or something similar and put a link in readme ?
It would be very helpful for users without a big GPU

**Comment by zhr1201** [comment]
Thank you for your great advise. I am now not able to access my labs' server because it's Winter Vocation in China and I am not at my school. After the Spring Festival, I will upload my model. 

If you just have like less than 100 types of noise, I believe training the current model with Adam and tuning the learning rate a little bit will work fine. If you are dealing with a larger noise dataset, I believe you should probably adding more layers and using skip connections.

**Comment by anicolson** [comment]
Hi, I would also be interested in a trained model.
